addons/html_editor/static/tests/link/popover.test.js

```diff
@@ -142,6 +142,20 @@ describe("popover should switch UI depending on editing state", () => {
         expect(".o_we_edit_link").toHaveCount(1);
         expect(".o_we_remove_link").toHaveCount(1);
     });
+    test("changes to link text done before clicking on edit button should be kept if discard button is pressed", async () => {
+        const { editor } = await setupEditor(
+            '<p>this is a <a href="http://test.com/">link[]</a></p>'
+        );
+        await waitFor(".o-we-linkpopover");
```

Comment:
`await waitFor(".o-we-linkpopover", { timeout: 1500 });`
Sometimes the linkpopover is not rendered within the default timeout so we use a bigger value